### DISCUSSION TITLE
Amp support for app router

### DISCUSSION THREAD

**Discussion by ariannargesi** [vercel/next.js#63975] [App Router]
Hi everyone 
Does app router support AMP pages? 
I can't find detail in the documentation and I appreciate any help.
Thanks in advance.

**Comment by wernerpj1** [comment]
Hey im looking for this they said about export const config = { amp: true } but doesnt worked for me im using Blog Starter Kit  trying to undestand how to implement nextjs sais export const config = is deprecied ... Im lost thinking in change for worpress they have an easy pluggin

**Comment by rizkiandrianto** [comment]
Got an answer but not trying it yet. Creating an API that returning AMP HTML, sounds makes sense to me.
https://stackoverflow.com/q/76918567/4536845
